asreview/webapp/auth.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import os
+
+from flask_httpauth import HTTPBasicAuth
+from werkzeug.security import generate_password_hash, check_password_hash
+from getpass import getpass
+
+from asreview.entry_points.auth import _auth_parser
+from asreview.webapp.utils.paths import get_auth_file_path
+
+
+class ASReviewAuth(HTTPBasicAuth):
+    USERPWD_DELIMITER = '|'
+    ANONYMOUS_USERNAME = 'anon'
+
+    def __init__(self, auth_file=None, scheme=None, realm=None):
+        super().__init__(scheme=scheme, realm=realm)
+
+        self.reinit(auth_file)
+
+
+    def reinit(self, auth_file):
+        self.auth_file = auth_file
+
+        if not auth_file:
+            self.auth_file = get_auth_file_path()
```

Comment:
Here it would be nice to inform the user that we are retrieving the default "auth.txt" file

Reply:
Done